src/frontend/Komponenter/Behandling/Brev/BrevUtils.ts

```diff
@@ -74,35 +75,53 @@ export const initFlettefelterMedVerdi = (
             hentVerdiFraMellomlagerEllerNull(flettefeltFraMellomlager, flettefeltReferanse._id),
     }));
 
-export const initValgteFeltMedMellomlager = (
+export const initValgteFelt = (
     valgteFeltFraMellomlager: ValgtFelt | undefined,
-    brevStruktur: BrevStruktur
-): ValgtFelt =>
-    Object.entries(valgteFeltFraMellomlager || {}).reduce((acc, [valgfeltApiNavn, mulighet]) => {
-        const utledOppdaterteFlettefeltFraSanity = () =>
-            brevStruktur.dokument.delmalerSortert.flatMap((delmal) => {
-                const valgfelt = delmal.delmalValgfelt.find(
-                    (valgFelt) => valgFelt.valgFeltApiNavn === valgfeltApiNavn
-                );
+    brevStruktur: BrevStruktur,
+    valgfeltStore: ValgfeltStore
+): ValgtFelt => {
+    const valgfeltMellomlager = Object.entries(valgteFeltFraMellomlager || {}).reduce(
+        (acc, [valgfeltApiNavn, mulighet]) => {
+            const utledOppdaterteFlettefeltFraSanity = () =>
+                brevStruktur.dokument.delmalerSortert.flatMap((delmal) => {
+                    const valgfelt = delmal.delmalValgfelt.find(
+                        (valgFelt) => valgFelt.valgFeltApiNavn === valgfeltApiNavn
+                    );
 
-                const valgtValgmulighet = valgfelt?.valgMuligheter.find(
-                    (valgmulighet) => valgmulighet.valgmulighet === mulighet.valgmulighet
-                );
+                    const valgtValgmulighet = valgfelt?.valgMuligheter.find(
+                        (valgmulighet) => valgmulighet.valgmulighet === mulighet.valgmulighet
+                    );
 
-                const flettefeltFraSanity = valgtValgmulighet?.flettefelter || [];
+                    const flettefeltFraSanity = valgtValgmulighet?.flettefelter || [];
 
-                return flettefeltFraSanity;
-            });
+                    return flettefeltFraSanity;
+                });
 
-        return {
-            ...acc,
-            [valgfeltApiNavn]: {
-                ...mulighet,
-                flettefelter: utledOppdaterteFlettefeltFraSanity(),
-            },
-        };
+            return {
+                ...acc,
+                [valgfeltApiNavn]: {
+                    ...mulighet,
+                    flettefelter: utledOppdaterteFlettefeltFraSanity(),
+                },
+            };
+        },
+        {}
+    );
+
+    const automatiskeValgfelt = Object.entries(valgfeltStore).reduce((acc, [valgfelt, valg]) => {
```

Comment:
Hvordan blir dette hvis saksbehandler "overstyrer" valgfeltet, "mellomlagrer", går ut av løsningen og kommer tilbake?

🤔

Reply:
Det gir kanskje ikke så mye verdi på dette valget, men andre ikke-entydige valg vil kanskje ha det problemet...?

Reply:
Nå vil "automatiske valgfelt" trumfe mellomlagret versjon. Det er samme oppførsel vi har hatt på flettefeltene. I dette tilfellet kommer vi nok til å skjule/disable hele delmalen med valgfeltet så saksbehandler vil uansett ikke ha mulighet til å overstyre.

Men som du nevner må vi tenke litt på hvordan vi gjør det for ikke entydige valg. Ser for meg at vi i valgfeltStore legger til et felt, f.eks. `kanOverstyresAvSaksbehandler` som bestemmer hva som får presedens av mellomlager og valgfeltStore.